w3d_translator/validate.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+
+from errors import ValidationError
+
+
+def validate_project(dir: str):
+
+    # Path must be a directory
+    if not os.path.isdir(dir):
+        raise ValidationError(f"Error: Project is not a directory: {dir}")
+
+    # Project must contain an xml file
+    xml_files = list(
+        filter(lambda file: file.endswith(".xml"), os.listdir(dir))
+    )
+    if not len(xml_files):
+        raise ValidationError(
+            f"Error: Project does not contain an xml file: {dir}"
+        )
+
+
+def validate_in_multiple(dir: str):
+    # Path must be a directory
+    if not os.path.isdir(dir):
+        raise ValidationError(f"Error: IN_DIR is not a directory: {dir}")
+
+    # Folder should only contain subfolders
```

Comment:
should this be a hard requirement? or just only look inside dirs? what if there's a readme or similar?

Reply:
Right now they're caught in `w3d_translator.handleError()` so the message is printed but execution continues. Is there some sort of warning flag in Python? Or do you think I should just skip over anything that isn't a folder?

Reply:
I'd just skip anything that isn't in a folder

Reply:
We now only iterate through folder in `in_dir` 

```python
projects = [p for p in Path(in_dir).iterdir() if p.is_dir()]
```